### PR TITLE
[41942] Comment numbers are cut off on mobile

### PR DESCRIPTION
**Pull request by bsatarnejad**
Increase the value of right, so when the screen size is reduces, it won't be affected.

https://community.openproject.org/projects/openproject/work_packages/41942/activity
OP#42031

### CODE REVIEW

**Comment by bsatarnejad** [comment]
The test is red here on PR, but when i test it locally it is green.

File: frontend/src/global_styles/content/work_packages/tabs/_activities.sass [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 .comments-number
   position: absolute
-  right: 0
+  right: 5px
```
Comment (HDinger) [inline frontend/src/global_styles/content/work_packages/tabs/_activities.sass]: This affects the desktop screen sizes as well, resulting in the numbers being not aligned with the sub-headers any more:
Bildschirmfoto 2022-04-19 um 13 07 07

**Review by HDinger — CHANGES_REQUESTED** [review]
I am not quite sure whether changing the position of the number is the right approach here. It looks more like the whole content is a bit too far to the right. There is also a separate ticket for that, so maybe you can catch both errors in one run..
https://community.openproject.org/projects/openproject/work_packages/42031/activity

**Comment by HDinger** [comment]
Removing the the padding from the `content-wrapper` does't work either, because that results in some elements having no spacings at all:
Bildschirmfoto 2022-04-20 um 09 01 54


I am therfore closing this in favour of https://github.com/opf/openproject/pull/10521

**Pull request closed without merging**